scripts/loader

```diff
@@ -36,7 +36,7 @@ launch_kernel() {
   echo "JVM options: "${JVM_OPTIONS}
   echo "Nucleus options: "${OPTIONS}
   child_pid=""
-  trap 'echo Received SIGTERM; sigterm_received=1; kill -TERM ${child_pid}' TERM
+  trap 'echo Received SIGTERM; sigterm_received=1; kill -TERM ${child_pid}; wait ${child_pid}; echo Killed child PID' TERM
```

Comment:
Should there be a timeout here?

Reply:
Timeout would be applied from higher up, ie docker itself has a timeout before it just sends sigkill. Default for docker is 10s.

Reply:
+1
https://docs.docker.com/engine/reference/commandline/stop/